examples/run_squad.py

```diff
@@ -36,13 +36,11 @@
 
```

Comment:
Can you add Roberta to the title of the file (Finetuning the library models for .... )